### DISCUSSION TITLE
why the steam not found

### DISCUSSION THREAD

**Discussion by hasassnz** [akiver/cs-demo-manager#944] [General]
Steam already but not found . maybe can add a CS path function

**Comment by akiver** [comment]
Please create a bug report (not a discussion) with more details.

Just in case, **you need Steam to run CS**.

**Reply by hasassnz** [reply]
working 
![60691fc6a9e9f3a6f063379ee062e86](https://github.com/user-attachments/assets/cfb8e346-7319-4877-a59e-3f4c64d16f85)
![0883550a89c74a802a5011ed9d39f91](https://github.com/user-attachments/assets/d835a75a-ce98-4b0f-bcbc-efa46d1f89af)
![3240e2637ef180115f1f4575dc8b448](https://github.com/user-attachments/assets/e0d4dda7-353c-4184-8285-475df6da3510)

Win10 22H2
